manifests/install.pp

```diff
@@ -0,0 +1,13 @@
+# install.pp
+# Installs clamav package
+#
+
+class clamav::install {
+
+  package { 'clamav':
+    ensure  => installed,
+    name    => $clamav_package,
+    require => Anchor['clamav::begin'],
```

Comment:
Maybe we should drop this require, since the anchor is in the main class and not here.